### DISCUSSION TITLE
Can You delete yourself as a login user?

### DISCUSSION THREAD

**Discussion by HuanYu-Tan** [SELab-2/OSOC-2#424] [Polls]

**Comment by jay-tux** [comment]
Shouldn't we ask Miet?

**Reply by BramDevlaminck** [reply]
I'd say no, but we should definitely ask Miet

**Comment by KoenDesplenter** [comment]
I think that every platform is legally required to create a delete my account option. Just as every auto mail sender is required to create a unsubscribe from list option.

**Comment by Mouwrice** [comment]
We are already going to implement a delete account function in the user settings #393 , so why disallow it in the user page? We should however give out a warning before deleting your / an account.
